docs/metrics.md

```diff
@@ -58,67 +155,241 @@ at the current time, they should not be used to report historical data such as:
 Metrics are also ephemeral, meaning there is no guarantee that they will be persisted for any amount of time. If you need
 a way to view and analyze historical data, consider the [workflow archive](workflow-archive.md) or reporting to logs.
 
+### Counter, gauge and histogram
+
+The following terms have specific meanings for the metrics that we collect:
+
+- `counter`: A value that accumulates over time – you can think of this like an odometer on a car; it only ever goes up.
+    - The rate of change of counters can be a very powerful tool in understanding these metrics.
+- `gauge`: Measures a current value at the time it is read. An example would be the fuel gauge in a vehicle.
+- `histogram`: A client-side aggregation of values, such as request latencies. A histogram is a good choice if you are interested in value statistics. For example: How many requests take fewer than 1 second?
+
+Thanks to [OpenTelemetry](https://opentelemetry.io/docs/concepts/signals/metrics/#metric-instruments) for these explanations.
```

Comment:
```suggestion
These terms are [defined by OpenTelemetry](https://opentelemetry.io/docs/concepts/signals/metrics/#metric-instruments):

> - `counter`: A value that accumulates over time – you can think of this like an odometer on a car; it only ever goes up.
    - The rate of change of counters can be a very powerful tool in understanding these metrics.
> - `gauge`: Measures a current value at the time it is read. An example would be the fuel gauge in a vehicle.
> - `histogram`: A client-side aggregation of values, such as request latencies. A histogram is a good choice if you are interested in value statistics. For example: How many requests take fewer than 1 second?

```

This looks to be almost entirely a direct quote, so we should quote it. Though I would prefer to just link to it if we have no notable differences

Reply:
We have no notable differences, but I felt it was worth including directly as it's obviously been an area of confusion and I want to ensure users fully grasp the concepts.